services/repository/src/main/java/org/sagebionetworks/repo/web/service/AdministrationServiceImpl.java

```diff
@@ -105,35 +119,14 @@
 	@Autowired
 	TransactionSynchronizationProxy transactionSynchronizationManager;
 
-	/**
-	 * Spring will use this constructor
-	 */
-	public AdministrationServiceImpl(){
-		
-	}
-	/**
-	 * IoC constructor
-	 * 
-	 * @param backupDaemonLauncher
-	 * @param objectTypeSerializer
-	 * @param userManager
-	 * @param stackStatusManager
-	 * @param dependencyManager
-	 * @param messageSyndication
-	 */
-	public AdministrationServiceImpl(BackupDaemonLauncher backupDaemonLauncher,
-			ObjectTypeSerializer objectTypeSerializer, UserManager userManager,
-			StackStatusManager stackStatusManager,
-			MessageSyndication messageSyndication,
-			DBOChangeDAO changeDAO) {
-		super();
-		this.backupDaemonLauncher = backupDaemonLauncher;
-		this.objectTypeSerializer = objectTypeSerializer;
-		this.userManager = userManager;
-		this.stackStatusManager = stackStatusManager;
-		this.messageSyndication = messageSyndication;
-		this.changeDAO = changeDAO;
-	}
+	@Autowired
```

Comment:
Need to revert this pull request when we are done with this process.  Need to ensure these dependencies are removed from the services layer.